packages/test-utils/src/toMatchEntity.ts

```diff
@@ -71,10 +73,8 @@ export async function toMatchEntity<T>(actual: Entity, expected: MatchedEntity<T
     }
   }
 
-  // Blatantly grab `toMatchObject` from the guts of expect
-  const { getMatchers } = require("expect/build/jestMatchersObject");
```

Comment:
Was there a reason the `require` was being done in the method vs. as a top-level import?

Reply:
Ah hm, not that I recall :thinking: , probably just my at-the-time way of doing an import w/o knowing I could `@ts-ignore` it.